interfaces/devdoc/string_utils.md

```diff
@@ -0,0 +1,61 @@
+`string_utils` requirements
+============
+
+## Overview
+
+`string_utils` is a collection of string utils. The set of requirements in this document is far from complete.
```

Comment:
>collection of string utils [](http://example.com/codeflow?start=20&length=26)

consider a short list of what utilities we have right now, like:
- printf helpers
- wide/narrow conversions
- FILETIME to string helpers

Reply:
added